package.json

```diff
@@ -191,7 +190,7 @@
     "less-loader": "5.0.0",
     "license-webpack-plugin": "2.1.2",
     "loader-utils": "1.2.3",
-    "lodash": "^4.17.20",
+    "lodash.template": "~4.5.0",
```

Comment:
This is a leftover from previous commit 🙈

Reply:
Usage of lodash per-method-packages is discouraged and they will be removed in lodash v5 see https://lodash.com/per-method-packages

Reply:
@Phillip9587 good point 😃 tbh I shouldn't have touched this part of the code on this commit but I had issues compiling, as `lodash.template` is already used. Please open a new PR to figure this out 👍